### PR TITLE
docs: issue #210 Issue Template Updates for Figma link

### PR DESCRIPTION
**Pull request by aryanas159**
**Description**

This PR fixes #
Issue #210
**Notes for Reviewers**
I have copied the given information in the docs. I am happy to make any changes if required.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/ci.md [hunk 1]
```diff
@@ -22,5 +22,5 @@ assignees: ''
 - 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
 - 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
    - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
-- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/ci.md:25]: remove backticks, please.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -36,4 +36,4 @@ Steps to reproduce the behavior:
 The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
 - See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
 - See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
-
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:39]: remove backticks

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -19,3 +19,4 @@ assignees: ''
 The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
 - See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
 - See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
+- - 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/feature_request.md:22]: same here.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by aryanas159** [comment]
@leecalcote Hey! I have made the necessary changes, can you please review.

**Comment by leecalcote** [comment]
@yashchaddha, would you like to offer review?

**Comment by yashchaddha** [comment]
Yes @leecalcote

**Comment by leecalcote** [comment]
@yashchaddha and how does it look?

**Comment by yashchaddha** [comment]
Hi @leecalcote apologies for responding late, I was under the impression that I would be assigned as reviewer and then I should start. However I  have reviewed the above PR and looks good to me.

I wanted to confirm that this line has been added extra in bug_report.md and feature_request.md file. Is that fine? or shall we discard that because there will be two references to Meshery UI links, thus increasing redundancy.

Also @aryanas159  I would want you to squash your commits into one while opening a PR

Thankyou

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (yashchaddha) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: There is one "-" extra in line 22

- - 🎨 Wireframes

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (yashchaddha) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: 39th Line is added Extra, while could not find the same in Master branch. Kindly confirm with @leecalcote on this as the issue was to originally replace the Figma references

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (yashchaddha) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: 22nd Line is added Extra, while could not find the same in Master branch. Kindly confirm with @leecalcote on this as the issue was to originally replace the Figma references

Comment (aryanas159) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: @yashchaddha I was confused about this  too.

**Comment by aryanas159** [comment]
@leecalcote  Should I replace 
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

this description with 

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

or these are two different lines.

**Comment by leecalcote** [comment]
@yashchaddha guidance?

**Comment by yashchaddha** [comment]
Hi @aryanas159  The main idea is to replace the Figma references with the open invite link, hence instead of adding another line in feature_request and bug_report.md, Kindly replace the line 

- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

with

- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.


Thankyou

**Comment by aryanas159** [comment]
@yashchaddha I have made the required changes

**Comment by yashchaddha** [comment]
Lgtm, @aryanas159 can you squash these commits in one?

**Comment by aryanas159** [comment]
> Lgtm, @aryanas159 can you squash these commits in one?

@yashchaddha Done

**Comment by yashchaddha** [comment]
Hi @leecalcote this PR looks good to me and can be merged 

Thankyou

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -18,4 +18,5 @@ assignees: ''
 
 The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
 - See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
-- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
+- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/feature_request.md:21]:
```suggestion
- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). 
- Fill-in a [Community Member Form](https://layer5.io/newcomer) and join the [Community Slack](http://slack.meshery.io) for access.
```

File: .github/ISSUE_TEMPLATE/ci.md [hunk 5]
```diff
@@ -22,5 +22,5 @@ assignees: ''
 - 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
 - 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
    - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
-- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
 - 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/ci.md:26]:
```suggestion
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.meshery.io)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -35,5 +35,5 @@ Steps to reproduce the behavior:
 
 The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
 - See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
-- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
+- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:38]:
```suggestion
- See [Meshery site designs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9). 
- Fill-in a [Community Member Form](https://layer5.io/newcomer) and join the [Community Slack](http://slack.meshery.io) for access.
```

**Pull request merged**